maven-setup/src/main.ts

```diff
@@ -20,8 +29,20 @@ export async function run(): Promise<void> {
         // configure authentication
         const accessToken = await auth.getAccessToken(input.getDevelocityAccessKey(), input.getDevelocityTokenExpiry())
 
+        const downloadFolder = './maven-extensions'
```

Comment:
Could we rather use the work directory `maven.mavenBuildTool.getBuildScanWorkDir()`?